DigitalLearningSolutions.Web/Views/Shared/_SecondaryTabsNavMenu.cshtml

```diff
@@ -0,0 +1,23 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common
+@model TabsNavViewModel
+
+@{
+  var tabWidthClass = Model.TabLinks.Count() switch{
+    2 => "nhsuk-grid-column-one-half",
+    3 => "nhsuk-grid-column-one-third",
+    4 => "nhsuk-grid-column-one-quarter",
+    _ => "nhsuk-grid-column-full",
+    };
+}
+
+<nav class="hee-secondary-nav--tabs" aria-label="Secondary navigation tabs">
+  <div class="nhsuk-grid-row">
+    <ul>
+      @foreach (var link in Model.TabLinks) {
+        <li class="@(link.IsCurrentPage ? "active" : null) @tabWidthClass nhsuk-u-padding-0">
+          <a asp-controller="@link.AspController" asp-action="@link.AspAction" asp-all-route-data="link.AspAllRouteData">@link.LinkText</a>
+        </li>
+      }
+    </ul>
+  </div>

```

Comment:
Just noticed this in the knowledge sharing. Can the grid-row be put on the ul? 
![image](https://user-images.githubusercontent.com/59561751/161984283-f82c4b01-0933-4851-99af-aae2cae00392.png)

Reply:
That's strange, I would have though the row-column would have canceled out as usual. Looks OK now, so I guess its fine.